config/log.go

```diff
@@ -178,6 +180,13 @@ func otlpGRPCLogExporter(ctx context.Context, otlpConfig *OTLP) (sdklog.Exporter
 		if u.Scheme == "http" {
 			opts = append(opts, otlploggrpc.WithInsecure())
 		}
+		if otlpConfig.Certificate != nil {
```

Comment:
I'm curious about why this was put here and not the outer scope on line 190/191 as a sibling of `if otlp.Config.Compression != nil {` ? the other if statements in the inner block pertain to the result of `url.ParseRequestURI`, so if you think it's better to put otlpConfig checks in this block, it would be good to understand when to add them here and when to add them in the outer scope

Reply:
i ended up moving the code after thinking about this further. originally i thought it made sense to only have it in the case where an endpoint was configured, but after rethinking through it i moved it out at the same level as compression and the other parameters. figured if there was a problem with the cert config it would make sense to surface it regardless of what the endpoint configuration is